### PR TITLE
Period name change bug

### PR DESCRIPTION
**Pull request by akssinhaamit**
# Motivation and Context

# What has changed

# How to test?

# Links

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java [hunk 1]
```diff
@@ -105,6 +110,85 @@ public void createActionPlan_500Response() {
...
+
+    assertEquals(actionPlan.getId(), actionPlans.get(0).getId());
+  }
+
+  @Test(expected = HttpClientErrorException.class)
+  public void updateActionPlanNameAndDescription_500Response() {
+    // Given
+    when(restTemplate.exchange(
+            any(URI.class), eq(HttpMethod.PUT), any(HttpEntity.class), eq(ActionPlanDTO.class)))
+        .thenThrow(new HttpClientErrorException(HttpStatus.INTERNAL_SERVER_ERROR));
```
Comment (jterryons) [inline src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java:134]: a client error wont have a 500 response only 400s. An internal server error will throw a HttpServerErrorException not a HttpClientErrorException

File: src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java [hunk 2]
```diff
@@ -105,6 +110,85 @@ public void createActionPlan_500Response() {
...
+        .exchange(any(), eq(HttpMethod.GET), any(HttpEntity.class), eq(ActionPlanDTO.class));
+    assertNull(actionPlan);
+  }
+
+  @Test(expected = HttpClientErrorException.class)
+  public void getActionPlanByID_500Response() {
+    // Given
+    when(restTemplate.exchange(
+            any(URI.class), eq(HttpMethod.GET), any(HttpEntity.class), eq(ActionPlanDTO.class)))
+        .thenThrow(new HttpClientErrorException(HttpStatus.INTERNAL_SERVER_ERROR));
```
Comment (jterryons) [inline src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java:182]: Same for this test, you'll never get a 500 in a HttpClientErrorException

File: src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java (hunk 1 above)
Comment (akssinhaamit) [reply src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java]: done!

File: src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java (hunk 2 above)
Comment (akssinhaamit) [reply src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClientTest.java]: done

File: src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java [hunk 3]
```diff
@@ -172,6 +166,27 @@ public ActionPlanDTO getActionPlanBySelectorsSocial(String collectionExerciseId)
     return actionPlans.iterator().next();
   }
 
+  public ActionPlanDTO getActionPlanById(final UUID actionPlanId) {
```
Comment (jterryons) [inline src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java:169]: I cant see where this is used. whats calling this?

Comment (akssinhaamit) [reply src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java]: removed the unused client method.

**Pull request merged**